app/server/utils/transfer_filter.py

```diff
@@ -28,7 +29,13 @@ class TransferFilterEnum:
     DISCRETE        = "discrete"
     BOOLEAN_MAPPING = "boolean_mapping"
```

Comment:
side note: It'd be nice to formalise the syntax here a little in a later PR. Eg swap this to "[boolean]"